src/PKSim.Infrastructure/Workspace.cs

```diff
@@ -65,6 +65,8 @@ public void SaveProject(string fileFullPath)
          _workspacePersistor.SaveSession(this, fileFullPath);
          updateProjectPropertiesFrom(fileFullPath);
 
+         _project.HasChanged = false;
```

Comment:
this should only be done after project saved. After load, project might have changed if converted